wfl/cli/commands/calc.py

```diff
@@ -104,3 +106,22 @@ def atomization_energy(inputs, outputs, prop_prefix, prop, isolated_atom_info_ke
     )
 
 
+@click.command("quippy")
```

Comment:
Is this already under "descriptors" somehow?  If not, we should possibly move it under such a category, first because quippy also provides potentials, and 2nd, because people might want to calculate ACE descriptors some day.

Reply:
Ah yes, this is the reason for this renaming. Currently I've merged what was in "eval" and "descriptors" into "calc", because things in both categories could be "calculated" (@gabor1 pointed out). Didn't think about quippy/ace doing both, properties and descriptors. Maybe instead should rename "eval" -> "calc-prop" & "descriptors" -> "calc-desc" or "calc-properties" & "calc-descriptors"? Just `wfl descriptors` would be fine, but `wfl properties` is maybe not as clear? Or should we just keep `eval` and `descriptors`? (It's not a big issue, but I thought I'd touch on this in the same PR"

Reply:
So this is now "calc quippy"?  Can you briefly list the relevant part of the nested command tree structure that's available now (as of this PR)?

Reply:
yes; edited the PR description